pkg/oci/spec.go

```diff
@@ -115,22 +115,20 @@ func ApplyOpts(ctx context.Context, client Client, c *containers.Container, s *S
 	return nil
 }
 
+// defaultUnixCaps is the set of the default caps.
+// The following caps are disabled by default since containerd v2.0:
+// MKNOD, NET_RAW, SETFCAP, SYS_CHROOT, AUDIT_WRITE.
 func defaultUnixCaps() []string {
```

Comment:
(Alternatively, we may change the cap set only for the CRI plugin if it is too much breaking to change the defaultUnixCaps here)

Reply:
It's fairly breaking either way; the CRI plugin is used in a lot of systems.